### PR TITLE
Reg user setup

### PR DESCRIPTION
**Pull request by izaxveta**
Feature: 2 reviewers

### Ticket:
https://www.pivotaltracker.com/story/show/153691574

#### What does  this PR do?
Integrates registered user to log in and see appropriate flash message.
Will integrate functionality that limits views depending on user roles.

#### Where should the reviewer start?
spec/features/user/registered_user_can_sign_in_spec.rb

#### How should this be manually tested?
May want to generate `Role.create(name: 'Registered User')` within rails console and test out actual view in rails s if your heart desires.

#### What are the relevant story numbers?
153691574

#### Screenshots (if appropriate)
All tests passsing
image

#### Questions:
  - Do Migrations Need to be ran?
No
  - Do Environment Variables need to be set?
No
  - Any other deploy steps?
No

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -22,14 +22,14 @@ def verify_user
     if @user && @user.authenticate(params[:session][:password])
       login_successful
     else
-      flash[:failure] = "That login was unsuccessful"
+      flash[:failure] = "That login was unsuccessful."
```
Comment (marshallhouston) [inline app/controllers/sessions_controller.rb:25]: I'm not a huge fan of the language that came with this project for this flash message.

I'd prefer "Login unsuccessful." Not an essential change though.

**Review by marshallhouston — APPROVED** [review]
Nice work!

File: spec/features/user/registered_user_can_sign_in_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    expect(current_path).to eq('/login')
+
+    fill_in 'session[email]', with: "#{user.email}"
+    fill_in 'session[password]', with: "#{user.password}"
+
+    click_button 'Login'
+
+    expect(current_path).to eq('/dashboard')
+    expect(page).to have_content("Logged in as #{user.full_name}.")
```
Comment (anlewi5) [inline spec/features/user/registered_user_can_sign_in_spec.rb]: Just an ambiguity: You have `flash[:notice] = "Logged in as #{@user.first_name} #{@user.last_name}."` in the code, but here you're testing for `"Logged in as #{user.full_name}."`. Will these always match?

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -22,14 +22,14 @@ def verify_user
...
       redirect_to login_path
     end
   end
 
   def login_successful
     session[:user_id] = @user.id
-    flash[:notice] = "Logged in as #{@user.first_name} #{@user.last_name}"
+    flash[:notice] = "Logged in as #{@user.first_name} #{@user.last_name}."
```
Comment (anlewis) [inline app/controllers/sessions_controller.rb:32]: Can we use `@user.full_name` instead?

**Review by anlewis — APPROVED** [review]
Looks good! Left a note on some syntax cleaning, but no big issues that I can see.

**Pull request merged**